apps/web/src/components/HoverCard/HoverCard.tsx

```diff
@@ -144,6 +144,7 @@ const StyledCardHeaderContainer = styled(VStack)`
 
 const StyledLink = styled(Link)`
   text-decoration: none;
+  outline: none;
```

Comment:
this is to fix this and remove the blue outline
<img width="500" alt="Screenshot 2023-08-07 at 2 20 52 PM" src="https://github.com/gallery-so/gallery/assets/49758803/c4c878bf-f8a9-4ded-b259-92d998e7f354">